features/frameworks/greentea-client/source/test_env.cpp

```diff
@@ -72,6 +73,7 @@ void GREENTEA_SETUP(const int timeout, const char *host_test_name) {
 	char _value[48] = {0};
 	while (1) {
         greentea_parse_kv(_key, _value, sizeof(_key), sizeof(_value));
+        greentea_write_string("mbedmbedmbedmbedmbedmbedmbedmbed\r\n");
```

Comment:
Its not writing back. I have observed multiple times that at the start of test when we start reading serial output from the target. ```htrun``` missing few characters. it may received the ```__sync``` as ```{__sync;788234738012347239823}}``` observe the missing ```{``` character in the beginning. By printing this ```mbedmbed....``` it gives time in the beginning of the test. Pretty much the same reason why it is in the other direction.